client_test/crash_test.go

```diff
@@ -51,15 +57,18 @@ func TestCrashTolerance(t *testing.T) {
 		channelId := directlyFundALedgerChannel(t, clientA, clientB, types.Address{})
 
 		clientA.Stop()
+		storeA.Close()
```

Comment:
Once https://github.com/statechannels/go-nitro/pull/1145 we should update `client.Close` to call `store.Close`